### PR TITLE
Fix healtcheck fail when TELEGRAM_STAT is not set

### LINKED ISSUE #30: Fails if TELEGRAM_STAT is not set
For me, using docker, the container fails if the TELEGRAM_STAT variable isn't set.

Maybe because the healthcheck uses it but it's not available?

### PR DESCRIPTION
**Pull request by giuseppeM99**
This should close #30 
plus this removes curl from the dependencies

### CODE REVIEW

**Review by luckydonald — CHANGES_REQUESTED** [review]
I think we can get rid of `|| exit 1` for now.

File: Dockerfile [hunk 1]
```diff
@@ -30,6 +30,6 @@ RUN addgroup -g 101 -S telegram-bot-api \
  && chmod 700 ${TELEGRAM_TEMP_DIR}
 USER telegram-bot-api:telegram-bot-api
 
-HEALTHCHECK CMD curl -f http://localhost:8082/ || exit 1
+HEALTHCHECK CMD nc -z localhost 8081 || exit 1
```
[the remark this answers is not in the document]
Comment (giuseppeM99) [reply Dockerfile:33]: can you test it without `exit 1`? if that works i'll remove it [cc @MarcoBuster]

[the remark this answers is not in the document]
Comment (MarcoBuster) [reply Dockerfile:33]: @giuseppeM99, @luckydonald is right. 
```bash
marco@acer-pop:~$ nc 127.0.0.1 12345
marco@acer-pop:~$ echo $? 
1
```

**Comment by luckydonald** [comment]
I tested the necat command with that, and it works.
Tested on a Mac, but is linux-y enough, most of the time.

LGTM

**Review by luckydonald — APPROVED** [review]
LGTM

**Review by MarcoBuster — APPROVED** [review]

**Pull request merged**